2022-11-28 00:21:49,618 INFO    MainThread:9539 [wandb_setup.py:_flush():71] setting env: {}
2022-11-28 00:21:49,618 INFO    MainThread:9539 [wandb_setup.py:_flush():71] setting login settings: {}
2022-11-28 00:21:49,618 INFO    MainThread:9539 [wandb_init.py:_log_setup():371] Logging user logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/neurocoevolution_pytorch/wandb/run-20221128_002149-3etxbmlm/logs/debug.log
2022-11-28 00:21:49,619 INFO    MainThread:9539 [wandb_init.py:_log_setup():372] Logging internal logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/neurocoevolution_pytorch/wandb/run-20221128_002149-3etxbmlm/logs/debug-internal.log
2022-11-28 00:21:49,619 INFO    MainThread:9539 [wandb_init.py:init():404] calling init triggers
2022-11-28 00:21:49,619 INFO    MainThread:9539 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-11-28 00:21:49,619 INFO    MainThread:9539 [wandb_init.py:init():460] starting backend
2022-11-28 00:21:49,619 INFO    MainThread:9539 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-28 00:21:49,631 INFO    MainThread:9539 [backend.py:ensure_launched():216] starting backend process...
2022-11-28 00:21:49,638 INFO    MainThread:9539 [backend.py:ensure_launched():221] started backend process with pid: 9604
2022-11-28 00:21:49,640 INFO    MainThread:9539 [wandb_init.py:init():469] backend started and connected
2022-11-28 00:21:49,643 INFO    MainThread:9539 [wandb_init.py:init():533] updated telemetry
2022-11-28 00:21:49,660 INFO    MainThread:9539 [wandb_init.py:init():563] communicating current version
2022-11-28 00:21:50,822 INFO    MainThread:9539 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-11-28 00:21:50,822 INFO    MainThread:9539 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-11-28 00:21:50,973 INFO    MainThread:9539 [wandb_init.py:init():606] starting run threads in backend
2022-11-28 00:21:55,978 INFO    MainThread:9539 [wandb_run.py:_console_start():1810] atexit reg
2022-11-28 00:21:55,979 INFO    MainThread:9539 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-11-28 00:21:55,979 INFO    MainThread:9539 [wandb_run.py:_redirect():1689] Redirecting console.
2022-11-28 00:21:55,981 INFO    MainThread:9539 [wandb_run.py:_redirect():1745] Redirects installed.
2022-11-28 00:21:55,981 INFO    MainThread:9539 [wandb_init.py:init():633] run started, returning control to user process
2022-11-28 00:29:42,863 INFO    MainThread:9539 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-11-28 00:29:42,864 INFO    MainThread:9539 [wandb_run.py:_restore():1752] restore
2022-11-28 00:29:45,644 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-11-28 00:29:45,840 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-11-28 00:29:46,276 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 33702
}

2022-11-28 00:29:46,378 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 33702
}

2022-11-28 00:29:46,479 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 33702
}

2022-11-28 00:29:46,583 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 33702
  total_bytes: 33702
}

2022-11-28 00:29:46,687 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 33702
  total_bytes: 33702
}

2022-11-28 00:29:46,791 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 33702
  total_bytes: 33702
}

2022-11-28 00:29:46,896 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 33702
  total_bytes: 33702
}

2022-11-28 00:29:47,000 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 33702
  total_bytes: 33702
}

2022-11-28 00:29:47,347 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 33702
  total_bytes: 33702
}

2022-11-28 00:29:47,675 INFO    MainThread:9539 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 33702
  total_bytes: 33702
}
local_info {
}

2022-11-28 00:29:49,206 INFO    MainThread:9539 [wandb_run.py:_append_files():2180] logging synced files
